apps/music/elements/music-artwork.js

```diff
@@ -164,7 +165,7 @@ var template =
 
 proto.createdCallback = function() {
   var shadowRoot = this.createShadowRoot();
-  shadowRoot.innerHTML = template;
+  shadowRoot.innerHTML = Sanitizer.unwrapSafeHTML(template);
```

Comment:
Stupid question but... Can we call `unwrapSafeHTML()` above when we assign the template string to `template`? If so, that would mean that this sanitization would, at most, only ever happen once instead of every time the element is instantiated.